scss/core/_typography.scss

```diff
@@ -1,5 +1,10 @@
 @import "~bootstrap/scss/type";
 
+body {
```

Comment:
This is going to change our actual stylesheet in apps, right?

Reply:
Yes. Good catch, I should move this to the doc site css and release this change separately if desired.